src/schemas/puppet.ts

```diff
@@ -36,23 +41,24 @@ export interface PuppetRoomTopicEvent {
 }
 
 export const CHAT_EVENT_DICT = {
-  friendship  : 'document can be writen at here',
-  login       : 'document can be writen at here',
-  logout      : 'document can be writen at here',
-  message     : 'document can be writen at here',
-  'room-join' : 'document can be writen at here',
-  'room-leave': 'document can be writen at here',
-  'room-topic': 'document can be writen at here',
-  scan        : 'document can be writen at here',
+  friendship   : 'document can be written at here',
+  login        : 'document can be written at here',
+  logout       : 'document can be written at here',
+  message      : 'document can be written at here',
+  'room-invite': 'document can be written at here',
```

Comment:
Only keep this line.

We can ignore the code style format in this PR, for better readability of the diff.